README.md

```diff
@@ -277,7 +286,8 @@ This are just examples. `chmod` has a lot of different configurations for differ
 ### Extracting compressed files
 
 `tar -xvzf <file.tar.gz>` - used to extract the .tar.gz file<br>
-
+`gzip -d <file_to_decompress>` - used to extract the .gz file<br>
```

Comment:
Please add `gunzip <file_to_decompress>.gz` as well since it's very similar to this command